problem-1/problem-1.mojo

```diff
@@ -1,22 +1,15 @@
-from python import Python
+fn lengthOfLastWord(enterword: String) -> Int:
+    var count: Int = 0
 
-def lengthOfLastWord(enterword: String):
-    
-    py = Python.import_module("builtins")
-    
-    count = 0
-    for i in range(len(enterword)-1, -1, -1):
+    for i in range(len(enterword) - 1, -1, -1):
         if enterword[i] != " ":
             count += 1
         elif count > 0:
             break
-    
-    print("Length of the last word:")
-    print(count)
 
-
-def main():
-    input_string = "Hello World" 
-    lengthOfLastWord(input_string)
+    return count
 
 
+fn main():
+    for _ in range(100000):
+        _ = lengthOfLastWord("Hello World")
```

Comment:
You will need to write `keep(lengthOfLastWord("Hello World")`, otherwise `lenghOfLastWord` gets DCE'd.

```mojo
from benchmark import keep

...
```